airflow/www/static/js/dag/nav/FilterBar.tsx

```diff
@@ -49,9 +63,23 @@ const FilterBar = () => {
     transformArrayToMultiSelectOptions,
   } = useFilters();
 
+  // @ts-ignore
+  const isBaseDateDefault = moment(now).isSame(baseDate, "minute");
+  const isNumRunsDefault = numRuns === defaultDagRunDisplayNumber.toString();
+  const isRunStateDefault = !runState || !runState.length;
+  const isRunTypeDefault = !runType || !runType.length;
+  const areFiltersDefault =
+    isBaseDateDefault &&
+    isNumRunsDefault &&
```

Comment:
Why do you count the "number of runs" into the filter setting? It is contained in the filter bar but it is actually not really a filter, it rather provides more data/visibility?
I believe so far it has been included in the reset and therefore the inclusion into coloring would be consistent but I am just thinking if we should exclude the number of runs from filter and reset?

Reply:
That's right. it isn't a filter. Updated.